README.md

```diff
@@ -8,45 +44,45 @@ To make the best out of the material of this module, you will need:
 
 ## Setup
 
-**Install Pyhton and anaconda on your machine**  
+### Install Pyhton and anaconda on your machine 
 If you don't have them already installed, install Pyhton and Anaconda on your machine.
 Follow these instructions on [how to install anaconda](https://docs.anaconda.com/anaconda/install/)
 
-**Download the module folder**  
-Clic on Code/Download Zip at the top of the page.  
-Move the zipped folder in the directory of your choice and decompress it.  
-Open the terminal and navigate to the module directory.
-
-
-**Create a conda environment**  
+### Create a conda environment**
 Create a conda virtual environment with the name you prefer, then activate it to work within it.
-Run
+Open a terminal, or open 'Anaconda Prompt' from Anaconda Navigator, in there run:
 
 ```
 conda create --name env_name
 conda activate env_name
 ```
- 
-**Install the module requirements**
+
+
+### Download the module folder
+Clic on Code/Download Zip at the top of the page.  
+Move the zipped folder in the directory of your choice and decompress it.  
+Open the terminal and navigate to the module directory.
+
+### Install the module requirements
 
 Run in the terminal
 
 ```
 pip install -r requirements.txt
 ```
-**Open Jupyter**  
-You can now open the lesson's notebooks in your favourite editor, or just type:
 
+### Open Jupyter lab
+You can now open the lesson's notebooks in jupyter lab
 ```
-jupyter notebook
+jupyter lab
```

Comment:
* I suggest to teach with jupyter lab instead of jupyter notebook. It is now by default installed with anaconda, it is easier to use (both richer and simpler interface), you can more easily also edit other files, and for teaching it is also nicer (you have more overview of what you are doing and where you are in the folder structure). For teaching I also suggest to all use the same editor/jupyter lab (instead of letting students pick their favourite editor).